remoteworkitem/flatten_test.go

```diff
@@ -0,0 +1,22 @@
+package remoteworkitem
+
+import (
+	"encoding/json"
+	"testing"
+)
+
+func TestFlattenMap(t *testing.T) {
+	testString := []byte(`{"name":"shoubhik","assignee":{"1":"sbose","2":"pranav","participants":{"4":"sbose56","5":"sbose78"}},"name":"shoubhik"}`)
```

Comment:
You need to use resource.require here to specify when to run this test. See other tests for example.